index.html

```diff
@@ -3276,6 +3283,32 @@ <h2 id="canmakepayment-protections">
           opening multiple windows (tabs or pop-ups).
         </p>
       </section>
+      <section>
+        <h2 id="user-activation-requirement">
+          Lack of user activation requirement
```

Comment:
Let's remove the "Lack of" part? Keep it only as "User activation requirement"?

Reply:
Done